src/ydata_quality/drift/engine.py

```diff
@@ -135,12 +137,12 @@ def __test_model(self):
         if self.label:
             test_x = self.df.head().copy()
             test_x.drop(self.label, axis=1, inplace=True)
-            output = self.model(test_x)
-            assert isinstance(output, (Series, np.ndarray)), "The provided model failed to produce the expected output."
+            output = self.model(test_x)  # pylint: disable=not-callable
```

Comment:
you have to unwrap the optional, which means you have to do this
```python
model: ModelWrapper = self.model
output = model(test_x)
```

Reply:
solved by ca154de